### PR TITLE
feat: add component EditorJS

### PR DESCRIPTION
**Pull request by HyTekCoop**
The EditorJS component has been added to the project. Like a check it was connected to the Home page.

image

### CODE REVIEW

File: src/presentation/components/editorjs/Editor.vue [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import RawTool from '@editorjs/raw';
+import Embed from '@editorjs/embed';
+
+/**
+ * Inline Tools for the Editor
+ */
+import InlineCode from '@editorjs/inline-code';
+import Marker from '@editorjs/marker';
+
+export default {
```
Comment (neSpecc) [inline src/presentation/components/editorjs/Editor.vue:29]: please, use `<script setup>` syntax, it's more readable

File: src/presentation/pages/Home.vue [hunk 2]
```diff
@@ -2,12 +2,16 @@
   <div>
     <h1>Welcome to the Home Page</h1>
     <p>This is the home page content.</p>
+    <editor></editor>
```
Comment (neSpecc) [inline src/presentation/pages/Home.vue:5]:
```suggestion
    <Editor />
```

File: src/presentation/pages/Home.vue [hunk 3]
```diff
@@ -2,12 +2,16 @@
...
   </div>
 </template>
 
 <script lang="ts">
```
Comment (neSpecc) [inline src/presentation/pages/Home.vue:9]: use `script setup`

File: src/presentation/components/editorjs/Editor.vue
Comment (neSpecc) [inline src/presentation/components/editorjs/Editor.vue:1]: src/presentation/components/Editor.vue

Comment (HyTekCoop) [reply src/presentation/components/editorjs/Editor.vue:1]: Rename

File: src/presentation/pages/Home.vue (hunk 2 above)
Comment (HyTekCoop) [reply src/presentation/pages/Home.vue:5]: Rename

File: src/presentation/pages/Home.vue (hunk 3 above)
Comment (HyTekCoop) [reply src/presentation/pages/Home.vue:9]: Remove "export defualt" left only script setup

File: src/presentation/components/Editor/Editor.vue [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+import RawTool from '@editorjs/raw';
+import Embed from '@editorjs/embed';
+
+/**
+ * Inline Tools for the Editor
+ */
+import InlineCode from '@editorjs/inline-code';
+import Marker from '@editorjs/marker';
+
+const editor = new Editor({
```
[the remark this answers is not in the document]
Comment (slaveeks) [reply src/presentation/components/Editor/Editor.vue]: done

**Review by neSpecc — APPROVED** [review]

**Review by slaveeks — APPROVED** [review]

**Pull request merged**